src/docs/product/performance/transaction-summary.mdx

```diff
@@ -87,7 +87,7 @@ Frontend transactions will have a "Web Vitals" tab. Clicking on the tab will tak
 
 ### Key Transaction
 
-If you find an important transaction you want to return to, click "Key Transaction" in the upper right corner to star it. This will be saved so you can view this from your [Performance Homepage](/product/performance/). Keep in mind any key transaction you mark will not be shared with others in your organization and are limited to ten per user.
+If you find an important transaction you want to return to, click "Star for Team" in the upper right corner and mark it as a key transaction for any of your teams. This will be saved so you can view this from your [Performance Homepage](/product/performance/). The key transactions you mark will be shared with members of the same team and each team can have up to 100 key transactions.
```

Comment:
Definitely add in an EA note in this case as well. I've gone ahead and put it in both spots.

```suggestion
<Note>

This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>

If you find an important transaction you want to return to, click "Star for Team" in the upper right corner and mark it as a key transaction for any of your teams. This will be saved so you can view this from your [Performance Homepage](/product/performance/). The key transactions you mark will be shared with members of the same team and each team can have up to 100 key transactions.
```